### PR TITLE
adds tutorial-checklist

### PR DESCRIPTION
**Pull request by edublancas**
Added a new document with the tutorial/blog post checklist. For now, it only states that datasets must be publicly available.

----
:books: Documentation preview :books:: https://ploomber-contributing--54.org.readthedocs.build/en/54/

### CODE REVIEW

File: documentation/tutorial-checklist.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Tutorial checklist
+
+```{note}
+This guidelines also apply to [blog posts](https://ploomber.io/blog/)
+```
+
+## Sample data must be publicly available
+
+Datasets must be available publicly so we can download them inline. In other words, you should be able to get a URL that when opened should take you to the raw data ([example](https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv)). Furthermore, the data should be available in a third-party site and not uploaded to any of our repositories.
```
Comment (lfunderburk) [inline documentation/tutorial-checklist.md:25]: Wonder how we can incorporate the following links as examples

 https://archive.ics.uci.edu/ml/index.php

https://www.nyc.gov/site/tlc/about/tlc-trip-record-data.page

@edublancas  can add them along with a mini sample script to extract them via requests + pandas

Comment (lfunderburk) [reply documentation/tutorial-checklist.md:25]: Thoughts?

**Review by lfunderburk — COMMENTED** [review]
This is looking great! Thank you for getting this started.

What I am thinking of adding:

1. A mini script that can be reused to download the data, either via the code snippet you provided, or as a general solution .py file that can easily be reused with the suggested sites.
2. Taking over this PR and continue building on it.

Is it ok if I self assign?

**Comment by edublancas** [comment]
@lfunderburk: your suggestions make sense, please open a PR!

**Pull request merged**